iset.mm

```diff
@@ -161503,6 +161503,66 @@ According to Wikipedia ("Integer", 25-May-2019,
     ( ccnfld crg wcel cz cvv cmgp cfv cress co czring wceq cnring df-zring eqid
     zex mgpress mp2an ) ABCDECAFGZDHIJFGKLODAJRBEMRNPQ $.
 
+  ${
+    $d n x y B $.  $d f x y F $.  $d f n x y R $.  $d n .x. $.  $d n x y .1. $.
+    mulgghm2.m $e |- .x. = ( .g ` R ) $.
+    mulgghm2.f $e |- F = ( n e. ZZ |-> ( n .x. .1. ) ) $.
+    ${
+      mulgghm2.b $e |- B = ( Base ` R ) $.
+      $( The powers of a group element give a homomorphism from ` ZZ ` to a
+         group.  (Contributed by Mario Carneiro, 13-Jun-2015.)  (Revised by AV,
+         12-Jun-2019.) $)
+      mulgghm2 $p |- ( ( R e. Grp /\ .1. e. B )
```

Comment:
Hm, why is this "group element" represented by `.1.`? It seems that this theorems holds for any group element, not for the identity only...

Reply:
Since the theorem is all about addition, I don't know if I'd even call it the identity. I didn't write any of this or have a lot of ideas for rewriting it so my preference would be to add the sentence "The name ` .1. ` should not be taken as a constraint as it may be any group element." to the comment (in both set.mm and iset.mm) and leave it at that for now.

Reply:
> I just wanted to suggest to simply change `.1.` in `X`, but the I saw that ~mulgghm2 is nested within a block with `.1.` in a hypothesis, which is actually used for ring unities in other theorems of this block. This nesting is really annoying and should be resolved.

Yeah, exactly, when I started down that path I noticed the nesting, started wondering whether this actually gets used for anything other than a ring multiplicative identity, and ultimately figured it seemed like there were enough uncertainties that it probably isn't a good idea to try to combine any change with this pull request.